initializr/src/main/resources/templates/home.html

```diff
@@ -11,174 +11,167 @@
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
+                    <div class="form-group">
+                        <label for="packaging" class="control-label">${packagings.title}</label>
+                        <select class="form-control" id="packaging" name="packaging">
+                            <% packagings.content.each { %>
+                            <option value="${it.id}" ${it.default==true ?
+                            ' selected' : ''}>${it.name}</option>
                             <% } %>
                         </select>
                     </div>
-                </div>
-                <div class="form-group">
-                    <label for="language" class="col-md-3 control-label">${languages.title}</label>
-                    <div class="col-md-8">
+                    <div class="form-group">
+                        <label for="language" class="control-label">${languages.title}</label>
                         <select class="form-control" name="language" id="language">
                             <% languages.content.each { %>
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
+                            <option value="${it.id}" ${it.default==true ?
+                            ' selected' : ''}>${it.name}</option>
                             <% } %>
                         </select>
                     </div>
                 </div>
-                <div class="form-group">
-                    <label for="bootVersion" class="col-md-3 control-label">${bootVersions.title}</label>
-                    <div class="col-md-8">
-                        <select class="form-control" name="bootVersion" id="bootVersion">
-                            <% bootVersions.content.each { %>
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
-                            <% } %>
-                        </select>
+            </div>
+            <div class="row">
+                <h2>Quick add dependencies</h2>
+
+                <p>Add Spring Boot Starters and dependencies to your application</p>
+
+                <div class="form-group col-md-6 col-sm-12">
+                    <div class="form-group">
+                        <input id="autocomplete" class="form-control" type="text" placeholder="Search for Starters"
+                               name="autocomplete">
+                    </div>
+                    <div class="message">
+                        <div class="alert alert-warning invisible" role="alert">This dependency is not available for the
+                            chosen Boot version.
+                        </div>
                     </div>
+                    <p class="text-center">
+                        Or check out <a href="#starters-list">the complete list of Dependencies.</a>
```

Comment:
We should probably find a way to make this link more obvious to both newcomers (they don't know what they're looking for here) and power users (they should know when new dependencies have been added to the list).